app/lib/actions/katello/upstream_subscriptions/bind_entitlements.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Katello
+    module UpstreamSubscriptions
+      class BindEntitlements < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
```

Comment:
is KeepCurrentUser necessary? ( I have no idea what it actually does )

Reply:
I remember it from an issue in the past, but I can't remember what it does either. :) I know @jlsherrill knows.

Reply:
For the run and finalize phase by default there is no 'User.current'.  This middleware 'saves' the user during the plan phase, and sets it before the run and finalize phases.  It doesn't look like its needed here, but sometimes its hard to tell